### ISSUE TITLE
More logs ?

### ISSUE THREAD

**Issue by rgarrigue** [fguillot/kanboard#2022]
Hi

I'm working on a plugin, that I kickstarted by forking an existing one + sed "s;old name;new name;".

My kanboard is a fresh install using git clone / composer, sqllite. It worker so far as login admin/admin. Of course something went wrong as soon as I cloned my githib.com/rgarrigue/plugin-redmine-webhook into plugin folder, and I got a 

    Unable to load this plugin class \Kanboard\Plugin\plugin-redmine-webhook\Plugin

I enabled debug log in config.php. That's just giving me a bunch of 

    [2016-03-17 10:00:51] [debug] SQL_QUERIES=0
    [2016-03-17 10:00:51] [debug] RENDERING=0.020128011703491
    [2016-03-17 10:00:51] [debug] MEMORY=2.57M
    [2016-03-17 10:00:51] [debug] URI=/
    [2016-03-17 10:00:51] [debug] ###############################################

I enabled everything error-related in php.ini, set error level to E_ALL... nothing.

How can I get more intel for my dev?

**Comment by BlueTeck** [comment]
Did you check this? https://github.com/fguillot/kanboard/blob/master/doc/plugin-registration.markdown

For example "The first letter of the plugin name must be capitalized."

**Comment by rgarrigue** [comment]
I did read that doc. That unlikely to be the issue since I cloned plugin-github-webhook, all filename should be ok, for now I just renamed github|gitlab in redmine in the content.

I guess I'll find what's wrong with a bit more time spent on it.

But that's not my question. If kanboard is telling me he can't load the plugin, I'ld like to know what's wrong for him, hence more log. THat about efficiency for later on too :-)
